application/backend/src/main/resources/apps/typerefinery/web_root/main.css

```diff
@@ -155,4 +156,9 @@
 
 .d3Graph {
   width: 100%;
+}
+/* //new dashborad css*/
+.section.hl-page-section.hl-grid.hl-cols-12 {
+  align-items: flex-start;
+  margin-top: -67px;
```

Comment:
Did you check whether this does not affect any other components?

Reply:
Done